internal/hooks/auth_hooks.go

```diff
@@ -141,9 +146,8 @@ type CustomAccessTokenOutput struct {
 }
 
 type SendSMSInput struct {
-	UserID uuid.UUID `json:"user_id"`
-	Phone  string    `json:"phone"`
-	OTP    string    `json:"otp"`
+	User      *models.User `json:"user"`
```

Comment:
```suggestion
	User      *models.User `json:"user,omitempty"`
```

Reply:
Soudns good, let's add this. When will this be empty though?